src/views/shared/fiat-amount/fiat-amount.view.jsx

```diff
@@ -8,9 +8,9 @@ function FiatAmount({ amount, currency }) {
   const classes = useFiatAmountStyles();
 
   return (
-    <div className={classes.root}>
-      {CurrencySymbol[currency].symbol} {!isNaN(amount) ? amount : "--"}
-    </div>
+    <span className={classes.fiatAmount}>
```

Comment:
Why does this simple component needs the CSS rules `display`, `alignItems` and `justifyContent` on `classes.fiatAmount`?

I think we should remove them.